lib/ansible/modules/system/cron.py

```diff
@@ -512,28 +509,30 @@ def _read_user_execute(self):
         Returns the command line for reading a crontab
         """
         user = ''
+        cron_cmd = self.module.get_bin_path('crontab', required=True)
```

Comment:
@JeanFred @samdoran Note:  Since `get_bin_path()` has to search the filesystem for the program, it's better to do this once instead of multiple times per execution.

Since these functions are part of the CronTab class, you can do it in __init__() and save the value into an instance variable there.  Something like
``` python
class CronTab:
    def __init__(self, module, user=None, cron_file=None):
        self.cron_cmd = module.get_bin_path('crontab', required=True)
```

This is not a blocker for the backport but something to keep in mind.